<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conexion</class>
 <widget class="QDialog" name="Conexion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblTitulo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Conexion con el server</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPuerto">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Puerto</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="letPuerto">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>69</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="letIP">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblIP">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ip:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pboAceptar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>261</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Encender</string>
   </property>
   <widget class="QRadioButton" name="rdbApagar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Apagar</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbEncender">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Encender</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pboAceptar</sender>
   <signal>clicked()</signal>
   <receiver>Conexion</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
